### ISSUE TITLE
My mask and image are misaligned and not matching. What could be causing this issue?

### ISSUE THREAD

**Issue by phucbienvan** [facebookresearch/segment-anything#579]
My mask and image are misaligned and not matching. What could be causing this issue?
![image](https://github.com/facebookresearch/segment-anything/assets/78720117/87894e6a-70e3-4cb9-994e-44b5cfb2b8b0)

**Comment by deepakanand14** [comment]
@phucbienvan How did you solve this?

**Comment by phucbienvan** [comment]
@deepakanand14 
Here is the English translation of your request:

In the file: scripts/export_onnx_model.py, replace:
`dummy_inputs = {
    "image_embeddings": torch.randn(1, embed_dim, *embed_size, dtype=torch.float),
    "point_coords": torch.randint(low=0, high=1024, size=(1, 5, 2), dtype=torch.float),
    "point_labels": torch.randint(low=0, high=4, size=(1, 5), dtype=torch.float),
    "mask_input": torch.randn(1, 1, *mask_input_size, dtype=torch.float),
    "has_mask_input": torch.tensor([1], dtype=torch.float),
    "orig_im_size": torch.tensor([1500, 2250], dtype=torch.float),
}
`
with:
`height, width, channels = image.shape

dummy_inputs = {
    "image_embeddings": torch.randn(1, embed_dim, *embed_size, dtype=torch.float),
    "point_coords": torch.randint(low=0, high=1024, size=(1, 5, 2), dtype=torch.float),
    "point_labels": torch.randint(low=0, high=4, size=(1, 5), dtype=torch.float),
    "mask_input": torch.randn(1, 1, *mask_input_size, dtype=torch.float),
    "has_mask_input": torch.tensor([1], dtype=torch.float),
    # "orig_im_size": torch.tensor([1500, 2250], dtype=torch.float),
    # "orig_im_size": torch.tensor([500, 375], dtype=torch.float),
    # "orig_im_size": torch.tensor([1455, 970], dtype=torch.float),
    "orig_im_size": torch.tensor([width, height], dtype=torch.float),
}
`
Hope this helps you!
